variableHandling.go

```diff
@@ -0,0 +1,82 @@
+package task
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/BurntSushi/toml"
+	yaml "gopkg.in/yaml.v2"
+)
+
+var (
+	// VariableFilePath file containing additional variables
+	VariableFilePath = "Variables"
+)
+
+func (t Task) handleVariables() (map[string]string, error) {
+	localVariables := make(map[string]string)
+	for key, value := range t.Variables {
+		localVariables[key] = value
+	}
+	if fileVariables, err := readVariablefile(); err == nil {
+		for key, value := range fileVariables {
+			localVariables[key] = value
+		}
+	} else {
+		return nil, err
+	}
+	for key, value := range getEnvironmentVariables() {
+		localVariables[key] = value
+	}
+	return localVariables, nil
+}
+
+// ReplaceVariables writes variables into initial string
+func ReplaceVariables(initial string, variables map[string]string) string {
+	replaced := initial
+	for name, val := range variables {
+		replaced = strings.Replace(replaced, fmt.Sprintf("{{%s}}", name), val, -1)
```

Comment:
I think I'll replace this with [Go's template engine](https://golang.org/pkg/text/template/) after merge.

So we can have some variables assigned by default (like OS, Architecture, etc). And will allow users to program if's and range's. Example:

```yml
acmd:
  cmds:
    - {{if eq .GOOS "windows"}}windows-command{{else}}unix-command{{end}}
```

But don't change that in this PR, otherwise it'll become too big.